package.json

```
@@ -22,13 +22,16 @@
   "devDependencies": {
     "@netlify/plugin-lighthouse": "2.1.2",
     "@types/react": "17.0.13",
+    "autoprefixer": "^10.2.6",
     "eslint": "7.30.0",
     "eslint-config-next": "11.0.1",
     "husky": ">=6",
     "lint-staged": ">=10",
     "netlify-plugin-cache-nextjs": "1.6.1",
     "netlify-plugin-image-optim": "0.4.0",
+    "postcss": "^8.3.5",
     "prettier": "2.3.2",
+    "tailwindcss": "^2.2.4",
```

```suggestion
    "tailwindcss": "2.2.4",
```
Please remove all the caret characters to ensure that we're locking its version.

You can do so by running this command:

```
yarn add -DE autoprefixer@latest postcss@latest tailwindcss@latest
```